solidity/contracts/system/IKeepFactorySelector.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity 0.5.17;
+
+import {IBondedECDSAKeepFactory} from "@keep-network/keep-ecdsa/contracts/api/IBondedECDSAKeepFactory.sol";
+
+/// @title Keep factory selector.
+/// @notice Selects an appropriate keep factory for the given application.
```

Comment:
`for the given application.` We have this interface defined in tBTC and the strategy will be for tBTC-only. We do not need to be generic. Application is always tBTC in this context.